docs/workflow/tutorial-web-implement-third-party-qr-code-form-element.mdx

```diff
@@ -0,0 +1,114 @@
+---
+title: Implement a QR Code Custom Form Element using a Third Party Library
+description: Geocortex Workflow - Implement a QR code form element for web applications using a third party library
+---
+
+<!-- cspell:ignore qrcode -->
+
+import useBaseUrl from "@docusaurus/useBaseUrl";
+import WebPrereqs from "./snippets/prereqs-web.mdx";
+
+Implementing a custom [form element](key-concepts.mdx#form-elements) allows you to augment the existing [form elements that come with Geocortex Workflow](https://docs.geocortex.com/workflow/latest/help/Default.htm#wf5/help/form-elements/auto-complete.htm%3FTocPath%3DForms%7CForm%2520Element%2520Reference%7C_____0).
+
+This article will walk you through creating a form element that uses the [qrcode](https://github.com/soldair/node-qrcode) library to generate and display a QR code.
+
+<img src={useBaseUrl("img/workflow-web-custom-qr-code.png")} />
+
+## Prerequisites
+
+<WebPrereqs />
+
+## Overview
+
+Implementing a QR code custom form element for web applications using a third party library consists of the following steps:
+
+1. Adding third party library dependencies to your project.
+1. Creating the custom form element.
+1. Implementing the custom form element using the third party library.
+
+## Adding Dependencies
+
+1. Run `npm install qrcode` in the terminal to install the third party [qrcode](https://github.com/soldair/node-qrcode) library that generates QR codes.
+1. Run `npm install @types/qrcode` in the terminal to install type information for the third party library. This is optional, but it provides an improved developer experience.
+
+## Set up Custom Form Element Skeleton
+
+To create a new form element:
+
+1. Open the Workflow activity SDK in Visual Studio Code.
+1. Run `npm run generate` in the terminal.
+1. When prompted, select `Form Element`.
```

Comment:
Maybe you can merge these two (_when prompted_ and _enter the name_) the same way you did in the activity?

Reply:
Haha. I think the activity one is the odd one out. It doesn't tell you to select `Activity`.